packages/common/src/internal-non-workflow/tls-config.ts

```diff
@@ -35,3 +35,10 @@ export type TLSConfigOption = TLSConfig | boolean | undefined | null;
 export function normalizeTlsConfig(tls: TLSConfigOption): TLSConfig | undefined {
   return typeof tls === 'object' ? (tls === null ? undefined : tls) : tls ? {} : undefined;
 }
+
+export function defaultTLSFromApiKey(
```

Comment:
Why introduce a new function for this? Can't we simply pass a second argument to `normalizeTlsConfig`?